utils/S3UtilsMongoClient.js

```diff
@@ -261,6 +357,23 @@ class S3UtilsMongoClient extends MongoClientInterface {
             const retResult = this._handleResults(collRes, isVer);
             retResult.stalled = stalledCount;
 
+            if (inflightsPreScan > 0 && retResult && retResult.dataMetrics) {
+                if (Object.keys(retResult.dataMetrics.bucket).length > 0) {
+                    Object.keys(retResult.dataMetrics.bucket).forEach(key => {
+                        retResult.dataMetrics.bucket[key] = {
+                            ...retResult.dataMetrics.bucket[key],
```

Comment:
this test is not needed, can iterate on empty list (and avoids creating twice the list of keys... probably optimized, but we never know...)